source/mainnet/net/resources/release-notes.rst

```diff
@@ -11,6 +11,29 @@ Release Notes - Testnet
 
    To subscribe to updates on the Testnet status page click **Subscribe** to get all updates or click **Get updates** to choose to get all updates or only updates for specific products.
 
+January 12, 2023
+
+Concordium Node 5.1.3
+---------------------
+
+Concordium node version 5.1.3 introduces the following new features and improvements:
+
+- Improvements were made to allow greater concurrency with transaction processing. (Checking transaction signatures is done without acquiring the global state lock.)
```

Comment:
```suggestion
- Improvements were made to allow greater concurrency with transaction processing.
```
Some other checks are also done without the lock now. I would not go into those details.